test/e2e/tags.go

```diff
@@ -0,0 +1,45 @@
+package e2e
+
+// The use of ginkgo tags is encouraged for describing tests.
+// This file defines test tags and annotations.
+// Each annotation should have a corresponding description.
+
+type Tag string
+
+const (
+	// May fail in secured clusters which aren't specifically opened up to allow certain operations.
```

Comment:
Maybe we need something similar to `[Feature:...]` that describes downstream/deployment characteristics?  Something else that would into this domain would be tagging tests that assume SSH access to a given cluster.  Right now it's assumed to be tied to cloud provider, but it's an orthogonal concern.

eg: `[ClusterSupports:SSH]`, `[ClusterSupports:InsecureTmp]`